orca-pipelinetemplate/src/test/resources/templates/simple-001.yml

```diff
@@ -1,30 +1,36 @@
 schema: "1"
 id: simpleTemplate
 variables:
-- name: application
-  description: The application name
 - name: regions
   description: A list of regions to bake in
   type: list
 configuration:
   triggers:
   - type: jenkins
     master: spinnaker
-    job: SPINNAKER-package-{{ var.application }}
+    job: SPINNAKER-package-{{ application }}
     enabled: true
 stages:
 - id: bake
+  type: bake
   name: Bake
   config:
-    regions: "{{ var.regions }}"
-    package: "{{ var.application }}-package"
+    regions: |
+      [
+      {{#each regions}}
+      "{{ this }}"{{#unless @last}},{{/unless}}
+      {{/each}}
+      ]
```

Comment:
One thing I hadn't really thought about until writing the integration spec is that handlebar templates need to emit  JSON, _not_ YAML. I'm not sure if we want to support YAML directly, or if telling people to emit JSON is fine. Might cause some room for confusion, though.